cli/tests/testdata/run/test_and_bench_in_run.js

```diff
@@ -1,5 +0,0 @@
-Deno.test(function foo() {
-});
-
-Deno.bench(function bar() {
-});
```

Comment:
Why remove this test?

Reply:
because Deno.test is not available in `deno run`

Reply:
`Deno.test` should be available in `deno run` but it should an empty function that does nothing. Similarly for `Deno.bench`. This was a regression test.